components/dashboard/src/components/forms/TextInputField.tsx

```diff
@@ -97,7 +97,7 @@ export const TextInput: FunctionComponent<TextInputProps> = memo(
         return (
             <input
                 id={id}
-                className={classNames("w-full max-w-lg", className)}
+                className={classNames("w-full max-w-lg dark:text-[#A8A29E]", className)}
```

Comment:
- can we use a color that we use elsewhere?
- would this not be dependent on the `disabled` state?

Reply:
> can we use a color that we use elsewhere?

I used the SVG color of `Copy` icon. 

---

> can we use a color that we use elsewhere?
would this not be dependent on the disabled state?

Looks good to me, that both of the text is readable: 

![image](https://github.com/gitpod-io/gitpod/assets/55068936/fbf45ee2-de93-4d10-9b7e-a6e19cce0971)